Decisions.Equifax/PrequalificationOfOne/PrequalificationOfOneSteps.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Decisions.Equifax.ConsumerCreditReport.Request;
+using Decisions.Equifax.PreQualificationOfOne.Response;
+using DecisionsFramework;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.ServiceLayer;
+
+namespace Decisions.Equifax.PrequalificationOfOne
```

Comment:
There's only one step so the class name doesn't have to be plural and the class name and namespace aren't cased correctly.